io-package.json

```diff
@@ -103,16 +103,16 @@
     },
     "objects": [
         {
-            "_id": "_design/mqttclient",
+            "_id": "_design/custom",
             "language": "javascript",
             "views": {
                 "state": {
-                    "map": "function(doc) {if (doc.type == 'state' && doc.common.custom) emit(doc._id, doc.common);}"
```

Comment:
@GermanBluefox be really careful when change the custom funcs!! You might break redis logic!
objectsInRedis.js L2349ff!!!

Reply:
sure. and that custom was copied from history just means that it was not discobered as changed there ;-)